packages/mobile/src/identity/reducer.ts

```diff
@@ -73,11 +78,12 @@ const initialState: State = {
   e164NumberToAddress: {},
   e164NumberToSalt: {},
   askedContactsPermission: false,
-  isLoadingImportContacts: false,
-  contactMappingProgress: {
+  importContactsProgress: {
+    status: 0,
```

Comment:
Can you use the enum property name instead of the value?

Reply:
Great question. I tried that but it leads to uninitialized values because the enum isn't defined in this file.

Reply:
The root underlying problem is dependency cycles

Reply:
maybe create a new file just for the enum then?

Reply:
Okay I'll give that a try